packages/dcos-history/extra/dcos-history.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Mesos History: DC/OS Resource Metrics History Service/API
+Description=DC/OS History: caches and exposes historical system state
```

Comment:
I went with system to jive with datacenter operating system. Ideally the endpoint will be moved at some point to under `/system/v1/history/` instead of `/dcos-history-service/` like it is now.